Update CacheResponse.php

PubliAlex opened the pull request:
Only cache pages for anonymous users

PubliAlex committed:
Update CacheResponse.php

Only cache pages for anonymous users

FlusherDock1 commented on classes/middleware/CacheResponse.php:
```diff
@@ -59,6 +60,7 @@ protected function shouldCache(Request $request, $response)
         return $request->isMethod('GET')
             && $request->getQueryString() == null
             && $response->getStatusCode() == 200
+            && BackendAuth::getUser() == null
```
Use in this case BackendAuth::check()

PubliAlex committed:
Update CacheResponse.php

Update CacheResponse.php in favor of BackendAuth::check() instead of BackendAuth::getUser()

PubliAlex commented:
Updated the PR by following your advice

FlusherDock1 reviewed (changes requested):
`BackendAuth::check()` returns boolean value (true or false). Please change comparison from null to false. And i will merge it. Thanks

PubliAlex committed:
Update CacheResponse.php

Change null to false for BackendAuth::check()

PubliAlex commented:
Done :)
